core/src/main/resources/lib/form/rowSet.jelly

```diff
@@ -43,9 +43,11 @@ THE SOFTWARE.
       <d:invokeBody />
     </j:when>
     <j:otherwise>
-      <tr ref="${attrs.ref}" class="row-set-start row-group-start" style="display:none" name="${attrs.name}" />
-      <d:invokeBody />
-      <tr class="row-set-end row-group-end" />
+      <div ref="${attrs.ref}" class="row-set-start row-group-start tr" style="display:none" name="${attrs.name}" />
+      <div class="container tr">
+        <d:invokeBody />
+      </div>
+      <div class="row-set-end row-group-end tr" />
```

Comment:
div tags are not self-closing, I think this should be `<div class="row-set-end row-group-end tr"></div>`

Reply:
Also, while `<div>` tags are not self closing in html5, I don't think that's actually the proper way to consider this.
The language here isn't html5, it's actually xml. We aren't talking to the html5 parser, we're talking to the jelly parser which is xml aware (and validating, when I make xml mistakes, it screams at me).

Consider:
https://github.com/jenkinsci/jenkins/blob/1a27da9c7f0e566a1a9406081c0625a0ff8c2f57/core/src/main/resources/lib/form/repeatable.jelly#L164

-- beyond that, I don't have a particular horse in this race. i.e. It doesn't matter to me. (I'm still working through my actual problems w/ repeatable, and the various plugins.)